plugins/generator-1.21.1/neoforge-1.21.1/mappings/attributes.yaml

```diff
@@ -0,0 +1,34 @@
+_default: Attributes.ARMOR

```

Comment:
Please sort this list in accordance to order of the vanilla file, as it makes porting later much easier. Start with fields from Attributes and then from NeoForgeMod